<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="gb_disp">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.960409, y1:0.955, x2:0, y2:0.0227273, stop:0 rgba(159, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(0, 62, 0);</string>
      </property>
      <property name="title">
       <string>Отображение</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="rb_list1">
         <property name="text">
          <string>Радиокнопка 1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rb_list2">
         <property name="text">
          <string>Радиокнопка 2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QProgressBar" name="pgB_pgBar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);
selection-background-color: rgb(0, 85, 0);</string>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pb_pgBar">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>ProgressBar</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QComboBox" name="cbB_list">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
